locales/en/transit.json

```diff
@@ -964,5 +964,14 @@
             "LocationColor": "Location color",
             "PolygonColor": "Accessibility polygon default color"
         }
+    },
+    "popup": {
```

Comment:
maybe rename by definitionTooltip, to be consistent?

Reply:
Did documentationTooltip instead, to match the component name